### DISCUSSION TITLE
"Disable" panning when the zoom is below a specific level in cytoscape.js

### DISCUSSION THREAD

**Discussion by lhoupert** [cytoscape/cytoscape.js#2980] [Q&A]
Hi all,
I would like to "disable" panning when the zoom is below a specific level. 

I tried using `cy.panningEnabled(false)` but it also disable the zoom. (I just want to prevent panning but let the user being able to zoom).

I thought about doing this with a `return` in on("pan") event:

```javascript
    cy.on('pan', function (evt) { // don't allow panning 
        if (cy.zoom() < 1.5) {            
            console.log("Dont allow panning");
            return;
        }
    });
```

But it doesn't work, the panning action is still executed.

Any ideas?

I opened a stackoverflow question about it here: https://stackoverflow.com/questions/70787931/disable-panning-when-the-zoom-is-below-a-specific-level-in-cytoscape-js

@maxkfranz any idea?

**Comment by maxkfranz** [comment]
What would it mean to zoom *about a point* while having the panning immutable?
